classes/mail/mailables/UserCreated.php

```diff
@@ -79,7 +82,7 @@ public function recipients(User $recipient, ?string $locale = null): Mailable
     {
         $this->traitRecipients([$recipient], $locale);
         $this->addData([
-            static::$variablePassword => $recipient->getPassword()
+            static::$variablePassword => $this->password ?? $recipient->getPassword()
```

Comment:
It looks that User::getPassword() returns encrypted password. As you passing the original password to the constructor, you can remove `$recipient->getPassword()`.

Reply:
And we need to escape it with `htmlspecialchars()`

Reply:
Done .